web/src/pm_conversations.ts

```diff
@@ -1,18 +1,31 @@
 import {FoldDict} from "./fold_dict";
 import * as muted_users from "./muted_users";
 import * as people from "./people";
+import type {Message} from "./types";
 
-const partners = new Set();
+type PMConversationsParams = {
+    recent_private_conversations: {
+        max_message_id: number;
+        user_ids: number[];
+    }[];
+};
```

Comment:
A type alias whose only purpose is to declare the keyword parameters of a particular function should be inlined into that function.